src/frontend/src/components/public/CommentsSidebar/interfaces.ts

```diff
@@ -14,6 +15,7 @@ export interface Comment {
 }
 
 export default interface CommentSidebarProps {
+  pia: IPiaForm;
```

Comment:
just not sure here If we should delete piaId and use PiaId from pia. I keep it just not sure if removing it will  break anything.